building/build-debs/homeworld-admin-tools/resources/keyserver.yaml

```diff
@@ -178,7 +178,7 @@ grants:
   # CLIENT CERTIFICATES
 
   grant-kubernetes-worker:
-    group: kubernetes-nodes
+    group: nodes
```

Comment:
Since we are no longer using kubernetes-nodes, we should remove it from the group tree.